src/theme/mod.rs

```diff
@@ -452,10 +452,10 @@ mod customs_test {
                     exa: Some($exa.into()),
                 };
 
-                let mut meh = UiStyles::colourful(false);
-                let (result, _reset) = definitions.parse_color_vars(&vars, &mut meh);
-                assert_eq!(ExtensionMappings { mappings }, result);
-                assert_eq!($expected, meh);
+                let mut result = UiStyles::default();
+                let (_exts, _reset) = definitions.parse_color_vars(&mut result);
+                assert_eq!(ExtensionMappings { mappings }, _exts);
```

Comment:
Assuming we don't use reset, can't we do this:
```suggestion
                let (exts, _) = definitions.parse_color_vars(&mut result);
                assert_eq!(ExtensionMappings { mappings }, exts);
```